src/main/java/org/fcrepo/repository/FileFinder.java

```diff
@@ -69,6 +70,8 @@ public FileVisitResult visitFile(final Path file,
             final String uriRef = root.relativize(file).toString().replaceAll("\\.ttl$", "");
             LOGGER.info("Creating {} from {}", uriRef, filename);
             putter.put(uriRef, new FileEntity(file.toFile()));
+        } else if (filename.endsWith(".txt") && !filename.startsWith(".")) {
+
```

Comment:
I'm sorry my mind was going, this was from a previous attempt to load resources. I'll axe it.